fairlearn/metrics/__init__.py

```diff
@@ -10,14 +10,29 @@
 arguments may be present (most often sample weights), which will
 affect how the metric is calculated.
 
-The group metrics in this module have signatures
-``g(y_true, y_pred, group_membership, ...)``
-where ``group_membership`` is an array of values indicating
-a group to which each pair of true and predicted values belong.
-The metric is evaluated for the entire set of data, and also
-for each subgroup identified in ``group_membership``.
+This module provides the concept of a *grouped metric*.
```

Comment:
grouped metric -> disaggregated metric

Reply:
Changed. And in the single line headline at the start of the documentation block.